src/main/java/org/spongepowered/common/registry/type/scoreboard/CriteriaRegistryModule.java

```diff
@@ -45,6 +60,33 @@ public void registerDefaults() {
         register((Criterion) IScoreCriteria.PLAYER_KILL_COUNT);
         register((Criterion) IScoreCriteria.TOTAL_KILL_COUNT);
         register((Criterion) IScoreCriteria.DEATH_COUNT);
+        register((Criterion) IScoreCriteria.FOOD);
+        register((Criterion) IScoreCriteria.AIR);
+        register((Criterion) IScoreCriteria.ARMOR);
+        register((Criterion) IScoreCriteria.XP);
+        register((Criterion) IScoreCriteria.LEVEL);
+
+        for (Map.Entry<TextFormatting, SpongeTextColor> entry : TextColorRegistryModule.enumChatColor.entrySet()) {
+            final int colorIndex = entry.getKey().getColorIndex();
+
+            if (colorIndex < 0) {
+                continue;
+            }
+
+            final String id = entry.getValue().getId();
+
+            final Criterion teamKillCriterion = (Criterion) IScoreCriteria.TEAM_KILL[colorIndex];
+            register(teamKillCriterion);
```

Comment:
I realise it wasn't like this before, but can you prefix all instance methods and fields with `this.` please? Again, enforced for API 8, but I'd like to ensure that anything new going into API 7 also follows it so we're already in the swing of things!